mathesar/tests/api/test_constraint_api.py

```diff
@@ -72,54 +79,59 @@ def test_retrieve_constraint(create_table, client):
 
     response = client.get(f'/api/v0/tables/{table.id}/constraints/{constraint_id}/')
     assert response.status_code == 200
-    _verify_unique_constraint(response.json(), ['Case Number'], 'NASA Constraint List 3_Case Number_key')
+    _verify_unique_constraint(response.json(), constraint_column_id_list, 'NASA Constraint List 3_Case Number_key')
 
 
 def test_create_multiple_column_unique_constraint(create_table, client):
     table_name = 'NASA Constraint List 4'
     table = create_table(table_name)
-
+    columns = table.columns.all()
+    constraint_column_1 = columns[1]
+    constraint_column_2 = columns[3]
+    constraint_column_id_list = [constraint_column_1.id, constraint_column_2.id]
     data = {
         'type': 'unique',
-        'columns': ['Center', 'Case Number']
+        'columns': constraint_column_id_list
     }
     response = client.post(f'/api/v0/tables/{table.id}/constraints/', data=data)
     assert response.status_code == 201
-    _verify_unique_constraint(response.json(), ['Center', 'Case Number'], 'NASA Constraint List 4_Center_key')
+    _verify_unique_constraint(response.json(), constraint_column_id_list, 'NASA Constraint List 4_Center_key')
 
 
 def test_create_single_column_unique_constraint(create_table, client):
     table_name = 'NASA Constraint List 5'
     table = create_table(table_name)
-
+    constraint_column_id = table.columns.all()[3].id
     data = {
         'type': 'unique',
-        'columns': ['Case Number']
+        'columns': [constraint_column_id]
     }
     response = client.post(f'/api/v0/tables/{table.id}/constraints/', data=data)
     assert response.status_code == 201
-    _verify_unique_constraint(response.json(), ['Case Number'], 'NASA Constraint List 5_Case Number_key')
+    _verify_unique_constraint(response.json(), [constraint_column_id], 'NASA Constraint List 5_Case Number_key')
 
 
 def test_create_unique_constraint_with_name_specified(create_table, client):
     table_name = 'NASA Constraint List 6'
     table = create_table(table_name)
-
+    columns = table.columns.all()
+    constraint_column_2 = columns[3]
+    constraint_column_id_list = [constraint_column_2.id]
```

Comment:
Do you need this variable? It's only used once.

Reply:
It's used in multiple places, in line 123 and line 127